packaging/debian/rules

```diff
@@ -22,19 +22,28 @@ override_dh_auto_install:
 	install -p -m 0644 instance_configs.cfg debian/google-guest-agent/usr/share/google-guest-agent
 	install -d debian/google-guest-agent/lib/systemd/system
 	install -p -m 0644 gce-workload-cert-refresh.timer debian/google-guest-agent/lib/systemd/system/
+	if [[ -d google-guest-agent ]]; then\
+		install -p -m 0644 google-guest-agent/cmd/google_guest_agent/google_guest_agent debian/google-guest-agent/usr/bin/google_guest_agent_manager;\
+	fi
 
 override_dh_golang:
 	# We don't use any packaged dependencies, so skip dh_golang step.
 
 override_dh_auto_build:
 	dh_auto_build -O--buildsystem=golang -- -ldflags="-s -w -X main.version=$(VERSION)" -mod=readonly
+	if [[ -d google-guest-agent ]]; then\
+		make -C google-guest-agent cmd/google_guest_agent/google_guest_agent;\
+	fi
 
 override_dh_installinit:
 	# We don't ship sysvinit files or need script changes for them.
 
 override_dh_systemd_enable:
 	install -d debian/google-guest-agent/lib/systemd/system
 	install -p -m 0644 *.service debian/google-guest-agent/lib/systemd/system/
+	if [[ -d google-guest-agent ]]; then\
+		install -p -m 0644 google-guest-agent/build/configs/usr/lib/systemd/system/google-guest-agent.service debian/google-guest-agent/lib/systemd/system/google-guest-agent-manager.service;\
+	fi
 	install -d debian/google-guest-agent/lib/systemd/system-preset
 	install -p -m 0644 *.preset debian/google-guest-agent/lib/systemd/system-preset/
 	dh_systemd_enable google-guest-agent.service google-startup-scripts.service google-shutdown-scripts.service gce-workload-cert-refresh.timer
```

Comment:
we are missing `google-guest-agent-manager.service` here in the enable line.

Reply:
Done.